src/coffee/import.coffee

```diff
@@ -43,10 +43,12 @@ class Import
     options.importFormat = options.importFormat || "csv"
     options.csvDelimiter = options.csvDelimiter || ","
     options.encoding = options.encoding || "utf-8"
+    options.mergeCategoryOrderHints = !!options.mergeCategoryOrderHints
```

Comment:
Interesting - I found some info here: http://adripofjavascript.com/blog/drips/the-difference-between-boolean-objects-and-boolean-primitives-in-javascript.html

I will put there a `Boolean()` then..